app/views/staff/funds/show.html.haml

```diff
@@ -22,11 +22,19 @@
             = link_to @fund.organisation.name, organisation_path(@fund.organisation)
 
     .govuk-grid-column-one-third
+      .govuk-grid-row.activity
+        %h2.govuk-heading-m
+          = t("page_content.fund.activity")
+
+        - if @fund.activity
+          = link_to @fund.activity.identifier, fund_activity_path(@fund, @fund.activity)
+        - else
+          = form_tag(fund_activities_path(@fund), method: "post") do
+            = submit_tag t("page_content.fund.button.create_activity"), class: "govuk-button"
+  .govuk-grid-row
+    .govuk-grid-column-full-width
```

Comment:
The transaction table is out of alignment. I think the class name here can be `.govuk-grid-column-full` to fix it.